doc/policy/packages.md

```diff
@@ -117,3 +118,63 @@ backward-compatible for users and applications that rely on p2p transaction rela
+
+    1a. All of the directly conflicting transactions signal BIP125 replaceability explicitly.
+
+        *Rationale*: Any transaction that is replaceable without package RBF should still be
+        replaceable.
+
+    1b. All directly conflicting transactions have `nVersion=3` and each package transaction that
+        conflicts with a mempool transaction also has `nVersion=3`.
+
+        *Rationale*: This helps preserve the mempool policy that any transaction spending an
+        unconfirmed `nVersion=3` transaction must also have `nVersion=3`. Otherwise, the child of a
+        `nVersion=3` transaction could be replaced by a non-`nVersion=3` transaction, and thus there
+        would be an `nVersion=3` transaction in the mempool with a non-`nVersion=3` child.
+
+        *Rationale*: Since V3 is inherited, this form of RBF signaling is implicitly inherited. That is,
+        while a V3 transaction is unconfirmed, any descendant(s) accepted to the mempool must also be
+        V3 and are thus also replaceable.
+
+3. The child transaction does not conflict with any transactions in the mempool.
+
+*Rationale*: Reduce complexity. Package RBF is intended for scenarios in which the user cannot
+change the parent transaction (e.g. they were presigned with an untrusted party), but has control
+over the child. For example, Package RBF can be used to replace the commitment transaction of an LN
+counterparty by attaching a high-fee child to an anchor output. If the user later wants to update
+the child (e.g. a further fee-bump), that can be done using a single transaction.
+
+4. The minimum between package feerate and ancestor feerate of the child is not lower than the
+   individual feerates of all directly conflicting transactions and the ancestor feerates of all
+   original transactions.
+
+*Rationale*: Attempt to ensure that the replacement transactions are not less incentive-compatible
+to mine.
+
+5. The package fee pays an absolute fee of at least the sum paid by the original transactions.
+   The additional fees (difference between absolute fee paid by the package after deduplication and the
+   sum paid by the original transactions) pays for the package's bandwidth at or above the rate set
+   by the node's incremental relay feerate. For example, if the incremental relay feerate is 1
+   satoshi/vB and the package after deduplication contains 500 virtual bytes total, then the package
+   fees after deduplication is at least 500 satoshis higher than the sum of the original transactions.
+
+*Rationale*: These fee-related rules are identical to the replacement [policy for individual
+transactions](./mempool-replacements.md), treating the package (after deduplication) as one
+composite transaction. This works because all of the transactions in a
+child-with-unconfirmed-parents package are in the ancestor set of the child transaction.
+
+6. The number of original transactions does not exceed 100.
```

Comment:
replaced

Reply:
same as here: https://github.com/bitcoin/bitcoin/blob/master/doc/policy/mempool-replacements.md#current-replace-by-fee-policy

Would it be more clear if I change the terms, or repeat them in this doc?

Reply:
being explicit is always better imo